doc/contribute-to-core-lightning/coding-style-guidelines/writing-json-schemas.md

```diff
@@ -63,13 +63,14 @@ To add conditional fields:
    generate the Markdown syntax as `[*index* [*start*] [*limit*]]`.
 
 ## Generating Examples in Schema
-The `tests/autogenerate-rpc-examples.py` test script regenerates RPC examples for methods defined 
-in `doc/schemas/lightning-*.json`. These examples are located at the end of each schema page, 
-detailing `shell` and `json` request formats along with their corresponding `json` responses.
-The script utilizes the pytest suite to automate this task by running a test, `test_generate_examples`,
-that sets up test nodes, records RPC requests, and captures responses. Any new RPC command's examples 
-should also be included in this scripts. This test only executes example generation if `GENERATE_EXAMPLES=True`
-is set, preventing accidental overwrites from unrelated tests.
+The `tests/autogenerate-rpc-examples.py` test script regenerates RPC examples for methods defined
+in `doc/schemas/lightning-*.json`, if the environment variable `GENERATE_EXAMPLES` is set to 1.
+These examples are located at the end of each schema page, detailing `shell` and `json` request
+formats along with their corresponding `json` responses. The script utilizes the pytest suite to
+automate this task by running a test, `test_generate_examples`, that sets up test nodes, records
+RPC requests, and captures responses. Any new RPC command's examples should also be included in
+this scripts. This test only executes example generation if `GENERATE_EXAMPLES=True` is set,
```

Comment:
GENERATE_EXAMPLES=1 is what you actually used?  Even if =True works, nice to be consistent

Reply:
Fixed, thanks.